### PR TITLE
Updated ~script.py per #978

### PR DESCRIPTION
**Pull request by qckzr**
- Moved and renamed ~script.py to scripts/build_directory_md.py

- Added script, .github and requirements.txt to the list of ignore files in build_directory_md.py

- Updated DIRECTORY.md file.

### CODE REVIEW

**Comment by cclauss** [comment]
Please modify the final lines of __.travis.yml__ to run the script on each PR.

**Comment by qckzr** [comment]
Done cclauss@

File: scripts/build_directory_md.py [hunk 1]
```diff
@@ -51,8 +51,10 @@ def _markdown(parent, ignores, ignores_ext, depth):
 ignores = [".vs",
     ".gitignore",
     ".git",
-    "~script.py",
+    "script",
```
Comment (cclauss) [inline scripts/build_directory_md.py]: script or scripts?

Comment (qckzr) [reply scripts/build_directory_md.py]: Done

**Pull request merged**